Makefile

```diff
@@ -22,6 +22,10 @@ CORE_GENERATOR ?= rocketchip
 # soc generator doesn't actually do anything at all.
 SOC_GENERATOR ?= nop
 
+# The testbench for simulation
+# FIXME: what's the default?
+TESTBENCH ?= none
```

Comment:
I see. Maybe something like PATH_TO_TESTBENCH and give e.g. main/module1 if main.module1.testbench is the hierarchy?

Also, if only PrimeTime uses it for now, could we make it a PrimeTime specific argument, like https://github.com/ucb-bar/plsi/blob/bar/src/addons/core-generator/chisel/vars.mk#L12 ?